static/scss/profile.scss

```diff
@@ -58,16 +112,55 @@
     }
     .profile-form-row > .basic-info {
       width: 50%;
+      &.basic-info-full-width {
+        width: 100%;
+        padding: 20px 0;
+      }
+      &.basic-info-border-bottom {
+        border-bottom: 2px #E2E2E2 solid;
+        margin-right: 10px;
+        &.education-top-spacing {
+          margin-top: 17px;
+        }
+        &.employment-top-spacing {
+          margin-top: -11px;
+        }
+      }
       .school-type {
-        font-size: 12px;
-        font-weight: 400;
-        text-transform: lowercase;
+        margin-bottom: -10px;
+        font-size: 16px;
+        font-weight: 600;
+        text-transform: capitalize;
+      }
+      .profile-row-name {
+        font-size: 16px;
+        color: #404040;
+      }
+      .profile-row-date-range {
+        font-size: 16px;
+        font-weight: 300;
+        color: #b7b7b7;
+      }
+      .profile-row-new-line {
+        display: block;
+        width: 100%;
       }
     }
 
     .profile-card-header {
       > span {
-        font-size: 18px;
+        font-size: 22px;
+      }
+    }
+    .profile-card-header-controls {
+      text-align: right;
+      float: right;
+      display: inline;
+      > a {
+        color: #1B7AE3;
+        font-size: 19px;
+        font-weight: 400;
+        cursor: pointer
       }
     }
   }
```

Comment:
If you write css that's specific to the user page it should live in a separate file.